frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/auth.unit.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import { screen } from "__support__/ui";
+import {
+  createMockGroup,
+  createMockTokenFeatures,
+} from "metabase-types/api/mocks";
+import { setupGroupsEndpoint } from "__support__/server-mocks";
+import { setup, SetupOpts } from "./setup";
+
+const setupPremium = (feature: string, opts?: SetupOpts) => {
```

Comment:
@alxnddr activating different token features in the same test file doesn't work because plugins aren't reset after each test. If for some reason it currently works, try adding negative checks for something that shouldn't be present and you see the issue

Reply:
Currently I'm following the convention `[feature-flag-name].unit.spec.tsx` when we test for multiple features flags for in a single component, like here